be1-go/channel/lao/mod.go

```diff
@@ -477,6 +504,15 @@ func (c *Channel) processRollCallObject(action string, msg message.Message) erro
 	return nil
 }
 
+func createChirpingChannel(publicKey string, channelID string, hub channel.HubFunctionalities,
```

Comment:
this function should be on `(c *Channel)`, so you don't need to pass so many arguments